app/__init__.py

```diff
@@ -28,7 +28,9 @@ def create_app(test_config=None):
 
     db.init_app(app)
     migrate.init_app(app, db)
-
+    from .routes import tasks_bp
+    from .routes import goals_bp
```

Comment:
You can combine these to save a few lines or so
```suggestion
    from .routes import tasks_bp, goals_bp
```